lib/src/package_analysis/summary.dart

```diff
@@ -11,7 +12,18 @@ Future<PackageShape> summarizePackage(
   PackageAnalysisContext packageAnalysisContext,
```

Comment:
Just a suggestion to make your life easier :D
```suggestion
  PackageAnalysisContext context,
```